src/Playwright.Tests/BrowserTypeConnectTests.cs

```diff
@@ -50,7 +50,7 @@ public void SetUpAsync()
                     StartInfo =
                     {
                         FileName = GetExecutablePath(),
-                        Arguments = $"launch-server {BrowserType.Name}",
+                        Arguments = $@"launch-server {BrowserType.Name}",
```

Comment:
```suggestion
                        Arguments = $"launch-server {BrowserType.Name}",
```

Reply:
This was likely a conflict merge gone the wrong way -.-